app/data/sponsors.json

```diff
@@ -0,0 +1,18 @@
+{
+	"currentSponsors": [
+		{
+			"description": "Make email yours. Fast, private email that’s just for you.",
+			"href": "https://www.fastmail.com/phillyjs",
+			"imageHref": "/assets/logos/fastmail.png",
+			"name": "Fastmail",
+			"startDate": "May 2022"
```

Comment:
I'm kind of leaning toward leaving off the start date; it could be that this could put off potential sponsors who might want to pause a month.

Reply:
Okay makes sense. I've dropped this.